web/src/pages/lesson1/types/index.tsx

```diff
@@ -3,13 +3,7 @@ import { ReactElement } from "react";
 export default function Types(): ReactElement {
   return (
     <div>
-      Conor complete this page based off this&nbsp;
-      <a
-        target="_blank"
-        href="https://github.com/JustinGuedes/learn-coding/blob/main/lesson1/types.md"
-      >
-        page
-      </a>
+      Conor failed the assignment
```

Comment:
Why did you remove this?

Reply:
Because Conor didn't know what he was doing

Reply:
Checks out. 
![image](https://github.com/JustinGuedes/learn-coding/assets/91260404/8658c98e-1850-4e16-bf06-ccad5a5246a7)